spec/features/access_form_submission_page_spec.rb

```diff
@@ -2,8 +2,27 @@
 
 RSpec.feature "Access Form Submission Page", js: true do
 
+  before do
+    stub_request(:post, "https://et-api-example.com/v2/repondents/response").
```

Comment:
I guess you use this in a few places ?? If so, it might be worth putting it into a common helper ?

Reply:
Great point, thank you.

Refactored in 3a35222